pkg/scaling/modifiers/formula.go

```diff
@@ -53,6 +56,8 @@ func HandleScalingModifiers(so *kedav1alpha1.ScaledObject, metrics []external_me
 			log.Error(err, "error applying custom scalingModifiers.Formula")
 		}
 		log.V(1).Info("returned metrics after formula is applied", "metrics", metrics)
+	} else if len(fallbackMetrics) > 0 {
+		metrics = []external_metrics.ExternalMetricValue{fallbackMetrics[0]}
```

Comment:
if the fallback is true, we still have to return a single metric as `applyScalingModifiersFormula` does, but I'd create it from scratch to ensure that we return exactly a new metric for that, with all the expected values